website/docs/d/projects.html copy.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: projects"
+sidebar_current: "docs-mongodbatlas-projects"
+description: |-
+    Describes a Projects.
+---
+
+# mongodbatlas_projects
+
+`mongodbatlas_projects` describe all Projects. This represents a projects which will be applied to all clusters within the project.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_project" "test" {
+  name   = "project-name"
+  org_id = "<ORG_ID>"
+
+  teams {
+    team_id    = "5e0fa8c99ccf641c722fe645"
+    role_names = ["GROUP_OWNER"]
+
+  }
+  teams {
+    team_id    = "5e1dd7b4f2a30ba80a70cd4rw"
+    role_names = ["GROUP_READ_ONLY", "GROUP_DATA_ACCESS_READ_WRITE"]
+  }
+}
+
+data "mongodbatlas_project" "test" {}
+```
+
+## Argument Reference
+There is not arguments
+
+
+
+* `id` - Autogenerated Unique ID for this data source.
+* `total_count` - Represents the total of the projects
+
+* `results` - A list where each represents a Projects.
+
+
+### Project
+
+* `name` - The name of the project you want to create. (Cannot be changed via this Provider after creation.)
+* `org_id` - The ID of the organization you want to create the project within.
+
+* `teams.#.team_id` - The unique identifier of the team you want to associate with the project. The team and project must share the same parent organization.
+
+* `teams.#.role_names` - Each string in the array represents a project role assigned to the team. Every user associated with the team inherits these roles.
+The following are the valid roles and their associated mappings:
+
+* `GROUP_OWNER`
+* `GROUP_READ_ONLY`
+* `GROUP_DATA_ACCESS_ADMIN`
+* `GROUP_DATA_ACCESS_READ_WRITE`
+* `GROUP_DATA_ACCESS_READ_ONLY`
+
+
+
+See [MongoDB Atlas API](https://docs.atlas.mongodb.com/reference/api/project-get-all/) Documentation for more information.
```

Comment:
I would add another link here, https://docs.atlas.mongodb.com/reference/api/project-get-teams/ since we use both to get the info.  So change to:
See [MongoDB Atlas API - Project](https://docs.atlas.mongodb.com/reference/api/project-get-all/) and [MongoDB Atlas API - Teams](https://docs.atlas.mongodb.com/reference/api/project-get-teams/) Documentation for more information.